docs/vdc/networking-virtual-datacenter.md

```diff
@@ -0,0 +1,397 @@
+
+There are two major types of logs in Azure:
+
+-   [**Activity Logs**][ActLog] (referred also as "Operational Log") provide insight into the operations that were performed on resources in the Azure subscription. These logs report the control-plane events for your subscriptions. Every Azure resource produces audit logs.
+
+-   [**Azure Diagnostic Logs**][DiagLog] are logs generated by a resource that provide rich, frequent data about the operation of that resource. The content of these logs varies by resource type.
+
+[![9]][9]
+
+In a vDC, it is extremely important to track the NSGs logs, particularly this information:
+
+-   [**Event logs**][NSGLog]: provides information on what NSG rules are applied to VMs and instance roles based on MAC address.
+-   [**Counter logs**][NSGLog]: tracks how many times each NSG rule was executed to deny or allow traffic.
+
+All logs can be stored in Azure Storage Accounts for audit, static analysis, or backup purposes. When the logs are stored in an Azure storage account, customers can use different types of frameworks to retrieve, prep, analyze, and visualize this data to report the status and health of cloud resources.
+
+Large enterprises should already have acquired a standard framework for monitoring on-premises systems and can extend that framework to integrate logs generated by cloud deployments. For organizations that wish to keep all the logging in the cloud, [Log Analytics][../log-analytics/log-analytics-overview
+.md] is a great choice. Since Log Analytics is implemented as a cloud-based service, you can have it up and running quickly with minimal investment in infrastructure services. Log Analytics can also integrate with System Center components such as System Center Operations Manager to extend your existing management investments into the cloud.
+
+Log Analytics is a service in Azure that helps collect, correlate, search, and act on log and performance data generated by operating systems, applications, and infrastructure cloud components. It gives customers real-time operational insights using integrated search and custom dashboards to analyze all the records across all your workloads in a vDC.
+
+The [Network Performance Monitor (NPM)][NPM] solution inside OMS can provide detailed network information end-to-end, including a single view of your Azure networks and on-premises networks. With specific monitors for ExpressRoute and public services.
+
+#### Component Type: Workloads
+Workload components are where your actual applications and services reside. It's also where your application development teams spend most of their time.
+
+The workload possibilities are truly endless. The following are just a few of the possible workload types:
+
+**Internal LOB Applications**
```

Comment:
This list of workloads feels half-baked. Also this article doesn't seem like the place to introduce stuff like HDInsight and Event Hubs. Are there any high level considerations about different types of workload, that are specific to VDC? (e.g., internet facing vs intranet, or what kind of shared services might be needed, or something) -

Reply:
👍 to be addressed subsequently